src/cromshell/slim_metadata/command.py

```diff
@@ -0,0 +1,95 @@
+import logging
+
+import click
+
+from cromshell.metadata import command as metadata_command
+
+LOGGER = logging.getLogger(__name__)
+
+
+@click.command(name="slim-metadata")
+@click.argument("workflow_id")
+@click.option(
+    "-k",
+    "--keys",
+    help="Use keys to get a subset of the metadata for a workflow. "
+         "Separate multiple keys by comma (e.g. '-k id[,status,...]').",
+)
+@click.option(
+    "-des",
+    "--dont-expand-subworkflows",
+    is_flag=True,
+    default=True,
+    help="Do not expand subworkflow info in metadata",
+)
+@click.option(
+    "-x",
+    "--exclude_keys",
+    is_flag=True,
+    show_default=True,
+    default=False,
+    help="Toggle to either include or exclude keys that are specified "
+    "by the --keys option or in the cromshell config JSON.",
+)
+@click.pass_obj
+def main(
+    config,
+    workflow_id: str,
+    keys: list,
+    dont_expand_subworkflows: bool,
+    exclude_keys: bool,
+):
+    """Get a subset of the workflow metadata using default keys."""
+
+    LOGGER.info("slim-metadata")
+
+    # If no keys were provided then set key_param to empty else
+    # strip trailing comma from keys and split keys by comma
+    key_param = [] if not keys else str(keys).strip(',').split(',')
+
+    metadata_command.check_cromwell_server(config=config, workflow_id=workflow_id)
+
+    # Resolve and get metadata keys from cli, config file, or config default
+    metadata_parameter = resolve_and_return_metadata_keys(
+        cli_key=key_param,
+        cromshell_config_options=config.cromshell_config_options,
+        config_slim_metadata_default_param=config.SLIM_METADATA_PARAMETERS,
+    )
+
+    key_action = "include" if not exclude_keys else "exclude"
+    LOGGER.info("Metadata keys set to %s: %s", key_action, metadata_parameter)
+
+    metadata_command.obtain_and_print_metadata(
+        config=config,
+        metadata_param=metadata_parameter,
+        exclude_keys=exclude_keys,
+        expand_subworkflows=dont_expand_subworkflows,
+    )
+
+    return 0
+
+
+def resolve_and_return_metadata_keys(
+    cli_key: list,
+    cromshell_config_options: dict,
+    config_slim_metadata_default_param: list,
+) -> list:
+    """Determines which metadata keys to use from cli, config file, and default
+    parameters, then returns a string of the processed keys ready to be used
+    in an api call. Also returns true if default parameter is being used else false."""
+
+    # If keys is specified in cli then use this first
+    if cli_key:
+        LOGGER.info("Using metadata key(s) from command line options.")
+        return cli_key
+
+    # If metadata_keys is specified in cromshell config file then use it for keys
+    elif "slim_metadata_keys" in cromshell_config_options:
```

Comment:
can we move this magic string to  `config` module?

Reply:
In the last commit of the git actions PR there will be a dictionary listing all cromshell config parameters in the config module so we can use that here once it gets rebased